src/components/Trends/GetHypedTrends.js

```diff
@@ -2,18 +2,32 @@ import React from 'react'
 import { connect } from 'react-redux'
 import { compose, pure } from 'recompose'
 import * as actions from './actions.js'
+import { sortBy } from '../../utils/sortMethods'
 
 class GetHypedTrends extends React.Component {
   render () {
-    const { render, ...rest } = this.props
-    return render(rest)
+    const { render, items, ...rest } = this.props
+    const trends = items.slice(-1)
+    const props = {
+      ...rest,
+      items: trends.length > 0 ? [sortByHype(trends)] : trends
```

Comment:
This line breaks the logic of multiple trend snapshots. If `trends.length > 0 ?` is true, then `items` will always be a one item array

Reply:
oh, yes, now I understand for what was 3 arrays